### PR TITLE
Upgrade Xcode to 12.0

### PR DESCRIPTION
**Pull request by jkmassel**
### Description

Makes changes to the project required to support Xcode 12, including:

- Fixes an error when building caused by an old version of CocoaLumberjack
- Fixes a compilation failure
- Fixes many warnings about modular includes using quotes instead of angle brackets
- Updates CocoaPods to support Xcode 12

### CODE REVIEW

File: WordPressKit/WordPressOrgXMLRPCApi.swift [hunk 1]
```diff
@@ -51,12 +51,20 @@ open class WordPressOrgXMLRPCApi: NSObject {
         let sessionManager = Alamofire.SessionManager(configuration: sessionConfiguration)
 
         let sessionDidReceiveChallengeWithCompletion: ((URLSession, URLAuthenticationChallenge, @escaping(URLSession.AuthChallengeDisposition, URLCredential?) -> Void) -> Void) = { [weak self] session, authenticationChallenge, completionHandler in
-            return self?.urlSession(session, didReceive: authenticationChallenge, completionHandler: completionHandler)
+            guard let strongSelf = self else {
```
Comment (bjtitus) [inline WordPressKit/WordPressOrgXMLRPCApi.swift:54]: @jkmassel I think it's sufficient to not return the result here. The return type is `Void` anyway.

File: WordPressKit/WordPressOrgXMLRPCApi.swift [hunk 2]
```diff
@@ -51,12 +51,20 @@ open class WordPressOrgXMLRPCApi: NSObject {
...
+                return
+            }
+
+            return strongSelf.urlSession(session, didReceive: authenticationChallenge, completionHandler: completionHandler)
```
Comment (bjtitus) [inline WordPressKit/WordPressOrgXMLRPCApi.swift]: @jkmassel I think it's sufficient to not return the result here. The return type is `Void` anyway.

This means we don't need the `guard let strongSelf = self` above either.

Comment (jkmassel) [reply WordPressKit/WordPressOrgXMLRPCApi.swift]: Yep you're right!

Thanks @bjtitus!!

**Review by bjtitus — APPROVED** [review]
Build and ran using Xcode 12 and did not see any warnings.

:shipit:

**Comment by jkmassel** [comment]
Thanks @bjtitus!!

**Comment by shiki** [comment]
Thanks for fixing this!

**Pull request merged**